<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeInfoClass</class>
 <widget class="QDialog" name="NodeInfoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NodeInfo</string>
  </property>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>44</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>100</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Node Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Pos</string>
   </property>
   <widget class="QLineEdit" name="txtX">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>70</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>40</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtY">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>70</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>40</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y :</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
